pkg/controller/kfdef/kfdef_controller.go

```diff
@@ -189,6 +253,18 @@ func (r *ReconcileKfDef) Reconcile(request reconcile.Request) (reconcile.Result,
 		}
 		log.Infof("Deleting kfdef.")
 
+		// stop the 2nd controller
+		if len(kfdefInstances) == 1 {
+			close(stop)
+			b2ndController = false
+		}
+
+		// Uninstall Kubeflow
+		err = kfDelete(instance)
+		if err == nil {
```

Comment:
should we raise an error or at least a warning if delete failed?

Reply:
It makes sense to print out an error message.